src/components/forms/elements/input-element.tsx

```diff
@@ -22,7 +22,13 @@ const InputElement = ({ name, label, description, placeholder, type = 'text' }:
         <FormItem>
           <FormLabel>{label}</FormLabel>
           <FormControl>
-            <Input type={type} placeholder={placeholder} {...field} />
+            {
+              type === 'number' ? (
```

Comment:
Instead of converting like this, can we do the conversion `onSubmit`?

Reply:
if we do it there then we wont be able to enter string, because if the input type is text then it will convert that also to number and it won't work

Reply:
Owwh didn't understand. Maybe we can see this later

Reply:
i tried doing that, but its not working